metrix-mapping/src/main/java/com/powsybl/metrix/mapping/TimeSeriesMapperChecker.java

```diff
@@ -139,6 +137,19 @@ public void versionEnd(int version) {
         super.versionEnd(version);
     }
 
+    public static boolean isTwoWindingsTransformerWithOutOfBoundsPhaseTapPosition(Identifiable<?> identifiable, MappingVariable variable, int value) {
+        return identifiable instanceof TwoWindingsTransformer twoWindingsTransformer
+                && twoWindingsTransformer.hasPhaseTapChanger()
```

Comment:
Could you add a last test case where `twoWindingsTransformer.hasPhaseTapChanger()` is false? It's the last uncovered case :)